libs/components/forms/src/lib/modules/input-box/input-box.component.html

```diff
@@ -98,7 +98,7 @@
 
 <ng-template #inlineHelpTemplate>
   <sky-help-inline
-    *ngIf="helpPopoverContent || helpKey"
+    *ngIf="(helpPopoverContent || helpKey) && labelText"
```

Comment:
Input box is showing the inline help button when `labelText` is undefined.

Reply:
so input box had "easy mode" implemented first and none of its features were behind the label text opt in input. this would change how that would work, are we sure we want to do this?